packages/gatsby-theme-carbon/src/components/Meta.js

```diff
@@ -34,7 +34,9 @@ const Meta = ({ pageTitle, pageDescription, pageKeywords, titleType }) => {
           content: pageKeywords || keywords,
         },
       ]}
-    />
+    >
+      <html lang={lang || 'en'} />
```

Comment:
I think this is nice because even if someone doesn't add `lang` to their own metadata, at least their site will have a valid `lang` value 🤔 But let me know what you think!

Reply:
If they don't add one, it should grab the `en` you have defined in the theme's gatsby-config along with the other defaults (title, keyword etc)